Formula/c/chsrc.rb

```diff
@@ -0,0 +1,19 @@
+class Chsrc < Formula
+  desc "Change Source for every software on every platform from the command-line"
+  homepage "https://github.com/RubyMetric/chsrc"
+  url "https://github.com/RubyMetric/chsrc/archive/refs/tags/v0.1.5.tar.gz"
+  sha256 "212123e79c5b2331dfc994504964d457d1e224e4d4a107a0047a024c1a260028"
+  license "GPL-3.0-only"
+  head "https://github.com/RubyMetric/chsrc.git", branch: "main"
+
```

Comment:
also need livecheck as upstream has rolling latest tag.

Reply:
0.1.6 is marked as latest